packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,135 @@
+# Anchor Table Column HLD
+
+## Problem Statement
+
+We need to support links in table cells. We will do this by creating a new column type.
+
+## Links To Relevant Work Items and Reference Material
+
+[GitHub issue: Table allows configuration of hyperlink column type (#1012)](https://github.com/ni/nimble/issues/1012)
+
+## Implementation / Design
+
+This is a fairly straightforward execution of our custom column type pattern. The anchor column will use two fields from the data records: one for the link's visible text (the "label") and one for the url (the "href"). The column configuration will contain the rest of the native anchor parameters: `hreflang`, `ping`, `referrerpolicy`, `rel`, `target`, `type`, and `download`. The assumption is that these should be the same for each link of a given column (if used at all). An alternative would be to have a `*FieldName` property for each of the native anchor parameters, but that seems unlikely to be useful.
+
+If a table record is missing a href value a text span will be rendered rather than a link.
+
+The column will also take an optional `placeholder` value to use when a record does not define a label.
+
+-   _Element name_: `nimble-table-column-anchor`
+-   _Attributes/properties_:
+    -   `labelFieldName`
+    -   `hrefFieldName`
+    -   `placeholder`
+    -   `hreflang`
+    -   `ping`
+    -   `referrerpolicy`
+    -   `rel`
+    -   `target`
+    -   `type`
+    -   `download`
+
+### Cell Template
+
+We will conditionally render either a `nimble-anchor` or a `span` of text. If no label is provided, but we have a URL, we'll render a link with the URL itself as the label.
+
+```html
+When cellRecord.href present
+<nimble-anchor
+    href="${x => x.cellRecord.href}"
+    hreflang="${x => x.columnConfig.hreflang}"
+    ping="${x => x.columnConfig.ping}"
+    referrerpolicy="${x => x.columnConfig.referrerpolicy}"
+    rel="${x => x.columnConfig.rel}"
+    target="${x => x.columnConfig.target}"
+    type="${x => x.columnConfig.type}"
+    download="${x => x.columnConfig.download}"
+    underline-hidden
+    @mouseover="${(x, c) => setTitleWhenOverflow(...)}"
+    @mouseout="${(x, c) => removeTitle(...)}"
+>
+    ${cellState.cellRecord.label ?? cellState.cellRecord.href}
+</nimble-anchor>
+When cellRecord.href is missing
+<span
+    class="when cellRecord.label present, empty, otherwise 'placeholder'"
+    @mouseover="${(x, c) => setTitleWhenOverflow(...)}"
+    @mouseout="${(_x, c) => removeTitle(...)}"
+>
+    ${cellState.cellRecord.label ?? cellState.columnConfig.placeholder}
+</span>
+```
+
+As seen in the template, we use `mouseover` and `mouseout` handlers to conditionally set, then remove, the `title` attribute to provide a tooltip when text is trucated. This is the same pattern used by the text column type. Note that we set the `underline-hidden` attribute so that the text only gets an underline upon hover.
```

Comment:
```suggestion
As seen in the template, we use `mouseover` and `mouseout` handlers to conditionally set, then remove, the `title` attribute to provide a tooltip when text is truncated. This is the same pattern used by the text column type. Note that we set the `underline-hidden` attribute so that the text only gets an underline upon hover.
```